ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.2.yaml

```diff
@@ -9,6 +9,9 @@ build_root:
     name: release
     namespace: openshift
     tag: golang-1.12
+tag_specification:
+  name: "4.2"
+  namespace: ocp
```

Comment:
nit: In the master analog, you have `tag_specification` sorted alphabetically.  Here, you have it before `promotion`.  I dunno if there is a consensus around positioning, or any CI to enforce alphabetical ordering or anything, but seems like we should at least be consistent within this PR ;).

Reply:
Addressed. Thanks for catching but AFAIK, there is no consensus on positioning or enforcing alphabetical ordering, perhaps @stevekuznetsov knows it.